content-services/7.1/develop/rest-api-guide/folders-files.md

```diff
@@ -2206,6 +2208,20 @@ Adding aspects to a folder or file is a bit more complicated than just updating
 * [How to update metadata](#updatemetadatanode)  
 * [How to remove aspects](#removeaspectsnode) 
 
+>**Note:** It is not possible to include aspects or properties that are part of the `systemModel` (i.e. namespace `sys:`) in API calls.
+> 
+> For example, if you try and add the `sys:hidden` aspect to a node, the following error is returned: 
+> 
+>```{
```

Comment:
Hey, when built the formatting of this code block seems to look like:

{ “error”: { “errorKey”: “framework.exception.ApiDefault”, “statusCode”: 400, “briefSummary”: “NameSpace cannot be used by API: sys:hidden”, “stackTrace”: “For security reasons the stack trace is no longer displayed, but the property is kept for previous versions”, “descriptionURL”: “https://api-explorer.alfresco.com” } }

URL http://localhost:4000/content-services/latest/develop/rest-api-guide/folders-files/)

Should it be formatted like other code blocks?

Reply:
When a code block is inside a note it does not display in the usual way. I cannot do anything about it as far as I know

Reply:
This will be fixed by PR #721